test/functional/rpc_createmultisig.py

```diff
@@ -18,15 +18,23 @@
     assert_equal,
 )
 from test_framework.wallet_util import bytes_to_wif
+from test_framework.wallet import (
+    MiniWallet,
+    address_to_scriptpubkey,
+    getnewdestination
+)
 
 class RpcCreateMultiSigTest(BitcoinTestFramework):
     def set_test_params(self):
         self.setup_clean_chain = True
         self.num_nodes = 3
         self.supports_cli = False
+        if self.is_wallet_compiled():
+            self.requires_wallet = True
 
     def skip_test_if_missing_module(self):
-        self.skip_if_no_wallet()
+        if self.is_wallet_compiled() and not self.options.descriptors:
+            self.skip_if_no_bdb()
```

Comment:
Is this check still needed if you adjust the checks below to check for bdb instead?

Reply:
Changed the checks to check for bdb instead. Removed this check.